homeassistant/components/overkiz/climate_entities/__init__.py

```diff
@@ -8,27 +10,51 @@
 )
 from .atlantic_electrical_towel_dryer import AtlanticElectricalTowelDryer
 from .atlantic_heat_recovery_ventilation import AtlanticHeatRecoveryVentilation
+from .atlantic_pass_apc_heating_and_cooling_controlled_zone import (
+    AtlanticPassAPCHeatingAndCoolingControlledZone,
+)
 from .atlantic_pass_apc_heating_zone import AtlanticPassAPCHeatingZone
 from .atlantic_pass_apc_zone_control import AtlanticPassAPCZoneControl
 from .hitachi_air_to_air_heat_pump_hlrrwifi import HitachiAirToAirHeatPumpHLRRWIFI
 from .somfy_heating_temperature_interface import SomfyHeatingTemperatureInterface
 from .somfy_thermostat import SomfyThermostat
 from .valve_heating_temperature_interface import ValveHeatingTemperatureInterface
 
+
+@unique
+class Controllable(StrEnum):
+    """Enum for widget controllables."""
+
+    IO_ATLANTIC_PASS_APC_HEATING_AND_COOLING_ZONE = (
+        "io:AtlanticPassAPCHeatingAndCoolingZoneComponent"
+    )
+    IO_ATLANTIC_PASS_APC_ZONE_CONTROL_ZONE = (
+        "io:AtlanticPassAPCZoneControlZoneComponent"
+    )
+
+
 WIDGET_TO_CLIMATE_ENTITY = {
     UIWidget.ATLANTIC_ELECTRICAL_HEATER: AtlanticElectricalHeater,
     UIWidget.ATLANTIC_ELECTRICAL_HEATER_WITH_ADJUSTABLE_TEMPERATURE_SETPOINT: AtlanticElectricalHeaterWithAdjustableTemperatureSetpoint,
     UIWidget.ATLANTIC_ELECTRICAL_TOWEL_DRYER: AtlanticElectricalTowelDryer,
     UIWidget.ATLANTIC_HEAT_RECOVERY_VENTILATION: AtlanticHeatRecoveryVentilation,
-    # ATLANTIC_PASS_APC_HEATING_AND_COOLING_ZONE works exactly the same as ATLANTIC_PASS_APC_HEATING_ZONE
-    UIWidget.ATLANTIC_PASS_APC_HEATING_AND_COOLING_ZONE: AtlanticPassAPCHeatingZone,
     UIWidget.ATLANTIC_PASS_APC_HEATING_ZONE: AtlanticPassAPCHeatingZone,
     UIWidget.ATLANTIC_PASS_APC_ZONE_CONTROL: AtlanticPassAPCZoneControl,
     UIWidget.SOMFY_HEATING_TEMPERATURE_INTERFACE: SomfyHeatingTemperatureInterface,
     UIWidget.SOMFY_THERMOSTAT: SomfyThermostat,
     UIWidget.VALVE_HEATING_TEMPERATURE_INTERFACE: ValveHeatingTemperatureInterface,
 }
 
+# For Atlantic APC, some devices are standalone and control themselve, some others needs to be
+# managed by a ZoneControl device. Widget name is the same in the two cases.
+WIDGET_AND_CONTROLLABLE_TO_CLIMATE_ENTITY = {
+    UIWidget.ATLANTIC_PASS_APC_HEATING_AND_COOLING_ZONE: {
+        # Beavious seems to be the same as AtlanticPassAPCHeatingZone.
```

Comment:
```suggestion
        # Behaviour seems to be the same as AtlanticPassAPCHeatingZone.
```

Not sure what you tried to say here?

Reply:
I think this comment should be simply removed

Reply:
It should but the work is not finished yet on this because for your device @nyroDev there is a way to convert it to cooling and it's not handled by this PR, I may do another one for that but I lack someone with a working heat+cool device like yours